app/src/androidTest/java/com/github/se/polyfit/data/api/SpoonacularApiCaller.kt

```diff
@@ -0,0 +1,4033 @@
+import android.util.Log
```

Comment:
I would say in general in the test make sure for deserialization if the data we receive are not conform or what expected like null or "corrupt" are well tested for robustess